repo2docker/buildpacks/conda/freeze.py

```diff
@@ -58,43 +50,32 @@ def freeze(env_file, frozen_file):
                 return
     print(f"Freezing {env_file} -> {frozen_file}")
 
+    # FIXME: conda-lock 0.8 requires {platform} in template
+    # https://github.com/conda-incubator/conda-lock/pull/78
+    frozen_template = str(frozen_dest) + ".{platform}"
+    frozen_tempfile = pathlib.Path(frozen_template.format(platform=platform))
+
+    check_call(
+        [
+            "conda-lock",
+            # FIXME: adopt micromamba after ordering is fixed
+            # https://github.com/conda-incubator/conda-lock/issues/79
```

Comment:
https://github.com/conda-incubator/conda-lock/issues/79 is fixed, though to make review easier this could be done in a follow-up PR so it's clear if there are other changes